test/extended/openstack/loadbalancers.go

```diff
@@ -141,14 +144,115 @@ var _ = g.Describe("[sig-installer][Suite:openshift/openstack][lb] The Openstack
 		o.Expect(err).NotTo(o.HaveOccurred())
 		o.Expect(isLbMethodApplied(lbMethod, results, pods)).Should(o.BeTrue(), "%q lb-method not applied after 100 queries:\n%v\n", lbMethod, results)
 	})
+
+	for _, i := range availableLbProvidersUnderTests {
+		lbProviderUnderTest := i
+		// https://issues.redhat.com/browse/OCPBUGS-2350
+		g.It(fmt.Sprintf("should apply lb-method on UDP %s LoadBalancer when an UDP svc with monitors and ETP:Local is created on Openshift", lbProviderUnderTest), func() {
+
+			g.By("Checking cluster configuration")
+			cloudProviderConfig, err := getConfig(oc.AdminKubeClient(),
```

Comment:
Could this be added on the beforeeach? All the test cases seems to fetch it

Reply:
Sure! I will think about moving this to BeforeEach during the rebases as soon as https://github.com/openshift/openstack-test/pull/58 get merged